main.py

```
@@ -62,6 +114,14 @@ def leer_archivo(path: str) -> str:
     """Lee el contenido de un archivo dado su path."""
     global agent_log
     agent_log.append(f"→ Leyendo archivo: {path}")
+    
+    # SECURITY: Basic path validation to prevent directory traversal
+    import os.path
```

Ruff (0.11.9)

`os.path` imported but unused

Remove unused import: `os.path`

(F401)